documentation/bolt_running_plans.md

```diff
@@ -70,7 +70,15 @@ bolt plan run mymodule::myplan --targets app1.myorg.com --params $(@{load_balanc
 
 ## Specifying the module path
 
-In order for Bolt to find a task or plan, the task or plan must be in a module on the `modulepath`. By default, the `modulepath` includes `modules/` and `site-modules/` directories inside the Bolt project directory.
+In order for Bolt to find a task or plan, the task or plan must be in a module on the modulepath.
+The [Bolt project directory](#bolt_project_directories.md) is also loaded as a module, namespaced to the `project-name`
+specified in `bolt.yaml`. The default modulepath is `<Boltdir>/modules/:<Boltdir>/site-modules/`.
 
-If you are developing a new plan, you can specify `--modulepath <PARENT_DIR_OF/MODULE>` to tell Bolt where to load the module. For example, if your module is in `~/src/modules/my_module/`, run Bolt with `--modulepath ~/src/module`. If you often use the same `modulepath`, you can set `modulepath` in `bolt.yaml`.
+If you are developing a new task or plan you can create a `<Boltdir>/tasks/` or `<Boltdir>/plans/`
+directory to develop your content in, then run Bolt from the root of your Bolt project directory to
+test the task or plan.
```

Comment:
Does `<Boltdir>` here signify the name of the project directory? If so, I think it would cut down on a bit of confusion if we used something like `<PROJECT_NAME>` throughout instead. Unless I'm missing something, and this only applies to embedded project directories? If that's the case, let's remove the `<>`.